lib/SearchAndSort/readme.md

```diff
@@ -71,6 +71,7 @@ resultsCachedPosition | position object |  sets the `ItemToView` prop of the int
 resultsKey | string | Sets a `key` prop on the internally rendered `<MultiColumnList>`. Changing this value will re-initialize the MCL. If necessary, this can be used to refresh the component so that it resets/readjusts to updates in data. This should be used sparingly as it can cause multiple re-renders of the list.
 customPaneSubText | node | A component that will be rendered in PaneSubHeader instead of default.
 searchFieldButtonLabel | node | A component that will be rendered inside the SearchField button instead of default.
+`regExpForQuery` prop for replacing the value in SearchField from query.
```

Comment:
Please fill out this table completely and properly. The three fields are name, type, and description and are separated by a pipe, `|`. The description should provide information about when this prop should be used, i.e., when is the default insufficient? It isn't clear to me why would I want to replace the value in the search field. A sample value or two would be especially helpful.

Reply:
@zburke please check it out, I have updated the readme.